### ISSUE TITLE
pip install ?

### ISSUE THREAD

**Issue by BertrandBordage** [bryanhelmig/django-method-cache#1]
Hello!

Just a few silly questions from a newbie:
Do you plan to add this module to pip?  Besides, do you currently have a proper way to install it?
This works:
git submodule add [URL] my_project/

But it also puts the README.md of django-method-cache in my_project, which is quite a bad idea.

Cheers,
Bertrand

**Comment by bryanhelmig** [comment]
Hey Bertand, I haven't thought about doing much more with this library. I might just recommend copying the directory into your project and using it like any other app of your own (thats what I do).

I doubt there will be much more updating to the library, as there are other common libs out there as well (like https://bitbucket.org/kmike/django-cache-utils/).

**Comment by BertrandBordage** [comment]
Yes, I took a look at django-cache-utils before trying your library.
But django-method-cache was more seducing than django-cache-utils (it requires less code to set up).

Now, after testing both libraries, I abandonned the idea of caching methods.
In my application, caching methods took too much time compared to the amount of use of each cached method.
The solution for my particular case was to simply change my template architecture so that pages are tiny instead of being huge (this is what I call huge : http://criminocorpus.cnrs.fr/lois/45-174/2012-06-26/ [In less than one week it will become tiny]).

Thanks,
Bertrand
